src/main/java/org/folio/fqm/service/FqlToSqlConverterService.java

```diff
@@ -65,6 +65,31 @@ public class FqlToSqlConverterService {
   private static final String RANGED_UUID_TYPE = "rangedUUIDType";
   private static final String OPEN_UUID_TYPE = "openUUIDType";
   private static final String DATE_TYPE = "dateType";
+  public static final String ALL_NULLS = """
+          true = ALL(
+            SELECT(
+              unnest(
+                cast(
+                  %s
+                  as varchar[]
+                )
+              )
+            ) IS NULL
+          )
+          """;
```

Comment:
I spent too long trying to get this to work as a JOOQ statement, before eventually just settling on this beautiful monstrosity.